packages/hardhat-core/test/internal/hardhat-network/helpers/assertEqualTransactionMaps.ts

```diff
@@ -0,0 +1,238 @@
+import { assert } from "chai";
```

Comment:
What is this file testing exactly? It seems to create two helpers that aren't used outside of this file and then it tests them. Are they meant to be used in the future but aren't being used right now?

Reply:
The `assertEqualTransactionMaps` function is actually used in the tests for `TxPool` class right now. For instance [here](https://github.com/EthWorks/buidler/blob/transaction-pool/packages/hardhat-core/test/internal/hardhat-network/provider/TxPool.ts#L501).

Reply:
Oh, my bad, that file was not shown in the diff because it was too big and so I didn't catch it doing ctrl-f. Thanks!